src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
+        System.out.println(ball1.getRandomBall().toString());
+        Lottery ball2 = new Lottery();
+        System.out.println(ball2.getRandomBall().toString());
+        Lottery ball3 = new Lottery();
+        System.out.println(ball3.getRandomBall().toString());
```

Comment:
1. move to the separate method;
2. Class name is Lottery but the object name is a ball (It isn't obvious);
3. What I see here is you can create 1 object lottery, and then use a loop inside which calls lottery.getRandomBall().toString();
4. System.out.println() bad practice, try to use logs here: https://www.baeldung.com/java-logging-intro.